openflow13/nx_action.go

```diff
@@ -0,0 +1,953 @@
+package openflow13
+
+import (
+	"encoding/binary"
+	"errors"
+	"math"
+	"net"
+)
+
+const (
```

Comment:
Please add comments to these constant definitions.

Reply:
Done.